charts/steward/templates/deployment-tenant-controller.yaml

```diff
@@ -33,6 +33,10 @@ spec:
         image: {{ printf "%s:%s" .repository .tag | quote }}
         imagePullPolicy: {{ .pullPolicy | quote }}
         {{- end }}
+        args:
+        {{- if .Values.tenantController.args.logVerbosity }}
+        - {{ printf "-v=%d" ( .Values.tenantController.args.logVerbosity | int ) | quote }}
```

Comment:
Please add logVerbosity parameter in TenantController paramenters in the documentation. -> https://github.com/SAP/stewardci-core/blob/master/charts/steward/README.md#controllers

Reply:
done